src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/modifier/classmemberimpliedmodifier/InputClassMemberImpliedModifierObjNestedRecord.java

```diff
@@ -0,0 +1,17 @@
+/*
+ClassMemberImpliedModifier
+violateImpliedStaticOnNestedEnum = (default)true
+violateImpliedStaticOnNestedInterface = (default)true
+violateImpliedStaticOnNestedRecord = (default)true
+
+
+*/
+
+//non-compiled with javac: Compilable with Java16
```

Comment:
Please use lowest version of Java that allows this syntax.

Reply:
Done